src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java

```diff
@@ -48,6 +44,7 @@ private IridiumSkyblockAPI(@NotNull IridiumSkyblock iridiumSkyblock) {
      * @since 3.0.0
      */
     public static @NotNull IridiumSkyblockAPI getInstance() {
```

Comment:
Remove the NotNull annotation since it can be nullable now, probabbly add some sort of comment saying how its null if the plugin hasnt loaded yet?